docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,29 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_, it is quite big because it contains tho whole sysroot for the project in addition to some native tools, so depending on your internet connectivity it could take a couple of minutes.
```

Comment:
I don't think it will take anyone hours, if so then they need to fix their network. The other things I'll fix.